.eslintignore

```diff
@@ -1,14 +0,0 @@
-node_modules/
-
-build/
-custom_types/
-dist/
-docs/
-examples/
-generated/
-proto/
-types/
-
-# Type definitions are never written by hand in this repo. Even when checked-in
-# as a source file, it is auto-generated stuff following other projects' rules.
-*.d.ts
```

Comment:
There used to be `custom_types/` folders with hand written type definitions. But looking at

```
$ git clean -xdf 
$ find . -name "*.d.ts"
$ find . -name "custom_types"
```

this is not the case anymore